modules/core/include/merge_sort.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * @file contour_comparison.h
+ * @author Ben Zhang
```

Comment:
In general we put WARG as the author in the files and let git track individual authorship. While this does work for a file you've just created, it's annoying to maintain, especially since we have full change and author history automatically through git.